simexpal/schemes/experiments.json

```diff
@@ -171,6 +171,7 @@
 								"extensions": {"$ref": "#/definitions/str_list"},
 								"extra_args": {"$ref": "#/definitions/str_list"},
 								"set": {"$ref": "#/definitions/str_or_str_list"},
+								"subdir": {"type": "string"},
```

Comment:
Please document that this `subdir` is now available for in our [instances readthedocs documentation](https://simexpal.readthedocs.io/en/latest/instances.html).